dist/node-connector.js

```diff
@@ -31720,6 +31720,7 @@ module.exports =
 	exports.createResultError = createResultError;
 	exports.wrapMethod = wrapMethod;
 	exports.wrapWithErrorHandling = wrapWithErrorHandling;
+	/* eslint-disable consistent-this */
```

Comment:
What's the rationale for doing this instead of the granular line disables? This disables it for the whole rest of the file, and as a general practice can thus obscure errors.

Reply:
From discussion: due to prettier's introduction, the original line disables become more complicated. Suggestion to avoid covering the whole file: gate this with an `/* eslint-enable consistent-this */` below the last occurrence.

Reply:
so this is the `dist` built file.  will update the corresponding `src` file 😄